tests/legacy-cli/e2e/tests/generate/application/application-zoneless.ts

```diff
@@ -0,0 +1,9 @@
+import { ng } from '../../../utils/process';
+import { useCIChrome } from '../../../utils/project';
+
+export default async function () {
+  await ng('generate', 'application', 'app2', '--experimental-zoneless');
+  await useCIChrome('app2', 'projects/app2');
+  await ng('test', 'app2', '--watch=false');
```

Comment:
Looks like `ng test` is failing because `provideExperimentalZonelessChangeDetection` need to be added also added in `app.component.spec.ts`

Reply:
Okay @alan-agius4

Reply:
@alan-agius4 In each app.component.spec.ts that are in the assets folder do I need to add provideExperimentalZonelessChangeDetection() ?

Reply:
Yes, in https://github.com/angular/angular-cli/blob/c5bad578f9367fc705e5a3a6afcd89f944cc3966/packages/schematics/angular/application/files/standalone-files/src/app/app.component.spec.ts.template#L6 and https://github.com/angular/angular-cli/blob/c5bad578f9367fc705e5a3a6afcd89f944cc3966/packages/schematics/angular/application/files/module-files/src/app/app.component.spec.ts.template#L7

See: https://angular.dev/guide/experimental/zoneless#using-zoneless-in-testbed